.github/CONTRIBUTING.md

```diff
@@ -34,151 +30,191 @@ ensure that the community is free to use your contributions.
 
+We recommend installing **OpenBLAS**, which is our default choice:
 
-There are two ways of building Aer simulators, depending on our goal they are:
-1. Build Terra compatible addon.
-2. Build standalone executable
+    brew install openblas
 
-**Terra addon**
+The **CMake** build system will search for other **BLAS** implementation
+alternatives if **OpenBLAS** is not installed in the system.
+
+You further need to have **Xcode Command Line Tools** installed on macOS:
+
+    xcode-select --install
+
+There are two ways of building Aer simulators, depending on your goal:
+
+1. Build a Terra compatible add-on;
+2. Build a standalone executable.
+
+**Terra Add-on**
 
 For the former, we just need to call the ``setup.py`` script:
 
-```
-qiskit-aer$ python ./setup.py bdist_wheel
-```
+    qiskit-aer$ python ./setup.py bdist_wheel
 
-We are using `scikit-build <https://scikit-build.readthedocs.io/en/latest/>`_ as a substitute of `setuptools`.
-This is basically the glue between ``setuptools`` and ``CMake``, so there are various options to pass variables to ``CMake``, and 
-the undelying build system (depending on your platform). The way to pass variables is:
+We are using **scikit-build** as a substitute for **setuptools**. This is
+basically the glue between **setuptools** and **CMake**, so there are various
+options to pass variables to **CMake**, and the underlying build system
+(depending on your platform). The way to pass variables is:
 
-```
-qiskit-aer$ python ./setup.py bdist_wheel -- -DCMAKE_VARIABLE=Values -- -Makefile_or_VisuaStudio_Flag
-```
+    qiskit-aer$ python ./setup.py bdist_wheel -- -DCMAKE_VARIABLE=Values -- -
+    Makefile_Flag
 
 So a real example could be:
 
-```
-qiskit-aer$ python ./setup.py bdist_wheel -- -j8
-```
-    
-This is setting the CMake variable ``STATIC_LINKING`` to value ``True`` so CMake will try to create an statically linked cython
-library, and is passing ``-j8`` flag to the underlaying build system, which in this case is Makefile, telling it that we want to
-build in parallel, using 8 processes.
+    qiskit-aer$ python ./setup.py bdist_wheel -- -j8
 
-*N.B. on MacOS:*, you may need to turn off static linking and specify your platform name, e.g.:
+This is setting the **CMake** variable `STATIC_LINKING` to value `True` so
+**CMake** will try to create an statically linked **cython** library, and is
+passing `-j8` flag to the underlaying build system, which in this case is
+Makefile, telling it that we want to build in parallel, using 8 processes.
 
-```
-qiskit-aer$ python ./setup.py bdist_wheel --plat-name macosx-10.9-x86_64 -- -DSTATIC_LINKING=False -- -j8
-```
+> You may need to turn off static linking and specify your platform name, e.g.:
 
-After this command is executed successfully, we will have a wheel package into the ``dist/`` directory, so next step is installing it:
+>   qiskit-aer$ python ./setup.py bdist_wheel --plat-name macosx-10.9-x86_64 -- >   -DSTATIC_LINKING=False -- -j8
```

Comment:
This command has a typo, there should be no `>`, anyway let's change this and simplify, because STATIC_LINKING is disabled by default, so there's no need to specify it:
```
qiskit-aer$ python ./setup.py bdist_wheel --plat-name macosx-10.9-x86_64 -- -- -j8
```

Reply:
yeah, I messed up line breaks. Those > marks were meant to be markdown syntax to create a block quote.